app/controllers/message_handler_controller.rb

```diff
@@ -7,22 +8,15 @@ def handle
     # regardless of whether we're doing a `bundle exec` call or we're
     # getting the messages fed to us by SQS via POST to this HTTP endpoint
     # as is the standard for the Worker Tier in Elastic Beanstalk.
-    resp = QueueListener.new.perform(nil, params)
+    response = QueueListener.new.perform(nil, params)
 
     # Tell SQS everything went hunky dory and we can delete the message.
-    send_status_ok
+    render json: response, status: 200
```

Comment:
Does response actually get sent anywhere? Returning 200 should just pop the message off of the queue. Where's response supposed to be sent here and does it get there?